ACC2024/RobotDynamics/@RigidBody2D/test/getLagrangian.m

```diff
@@ -0,0 +1,22 @@
+function [L,T,V]= getLagrangian(bodyObjName,generalCoordinates,generalVelocities)
+% test the additional function that obtains the Lagrangian,L= T(kinetic
+% energy) - V(potential energy)
+% input: rigid body object;generalCoordinates;generalVelocities
+% output: L: Larangian; T: kinetic energy; V: potential energy
+
+% get COM position & velocity 
+bodyCOM_Postion = bodyObjName.getCOMPosition();
+bodyCOM_Vel = bodyObjName.getCOMVelocity(generalCoordinates, generalVelocities);
+
+% get kinetic energy
+translationKineEnergy = bodyObjName.getCOMTransitionKineticEnergy(generalCoordinates, generalVelocities);
+rotationKineEnergy = bodyObjName.getRotationEnergy(generalCoordinates, generalVelocities);
+
+T = simplify(translationKineEnergy + rotationKineEnergy);
```

Comment:
There is a function in RigidBody2D that returns kinetic energy which is the sum of these two.